javalanglib/src/main/scala/java/lang/Thread.scala

```diff
@@ -39,10 +37,17 @@ class Thread private (dummy: Unit) extends Runnable {
     StackTrace.getCurrentStackTrace()
 
   def getId(): scala.Long = 1
+
+  // Javadocs:
+  // Returns the handler invoked when this thread abruptly terminates due to an uncaught exception. If this thread has not had an uncaught exception handler explicitly set then this thread's ThreadGroup object is returned, unless this thread has terminated, in which case null is returned.
+  // Therefore:
+  // JS VMs are running on a single thread, if this thread has been terminated it has to be this one, so, since it's terminated, we return null
+  def getUncaughtExceptionHandler(): Thread.UncaughtExceptionHandler = null
```

Comment:
I don't think this is correct. Even with a single thread, while the thread is running, `getUncaughtExceptionHandler()` already returns an instance of `ThreadGroup`:
```scala
>scala
Welcome to Scala 2.13.1 (Java HotSpot(TM) 64-Bit Server VM, Java 1.8.0_60).
Type in expressions for evaluation. Or try :help.

scala> val t = Thread.currentThread
t: Thread = Thread[main,5,main]

scala> t.getUncaughtExceptionHandler
res6: java.lang.Thread.UncaughtExceptionHandler = java.lang.ThreadGroup[name=main,maxpri=10]
```